adapters/factMapper/builder_test.go

```diff
@@ -26,17 +26,20 @@ func TestBuilderInvariants(t *testing.T) {
 	testutil.TestBuilderInvariants(b, t)
 }
 
-func TestNoRules(t *testing.T) {
+func setup(t *testing.T, bc BuilderConfig, ac AdapterConfig) adapters.FactConverter {
```

Comment:
what about having setup return `(adapters.FactConverter, error)` and then leaving the error handling and logging in the actual tests? That would avoid passing in `*testing.T` and keep the test bits in the body of the test methods?